### ISSUE TITLE
Lerna is not actually part of your devDependencies 

### ISSUE THREAD

**Issue by ghiscoding** [wixplosives/sample-monorepo#503]
Hi there,

Great monorepo sample, I noticed that the `package.json` does not actually reference Lerna anywhere even though you're mentioning it in the readme, I know that it can work with just `npx lerna` but it's probably better to include it in the `package.json`

Also another note that I came across your repo mainly because of a comment you recently posted on a recent PR of Lerna [v5.2.0](https://github.com/lerna/lerna/releases/tag/v5.2.0) which is now installing Nx even if you don't want it and I'm agreement with you there. I was afraid that the project now being directed by Nrwl would eventually have them push their own products. It's sad to see they're going all in with that direction, especially for users who might prefer or already have other tools configured like TurboRepo. On the good side there are alternatives, you can take a look at [Lerna-Lite](https://github.com/ghiscoding/lerna-lite) that I created few months before Nrwl took over stewardship of Lerna, at first I thought it was a bad timing from my side to create it before Lerna came back alive, but now that I see the direction they're going, I'm actually glad that I created Lerna-Lite which I use in my own monorepos. On the other end I did add their `useNx` option but it's totally and will remain optional and that's the way it should always be. 

So long story short, you can read all about it on why I created [Lerna-Lite](https://github.com/ghiscoding/lerna-lite), and there's extra features like the support of yarn/pnpm [`workspace:`](https://pnpm.io/workspaces#workspace-protocol-workspace) protocol and other extras that Lerna doesn't yet have. I also keep in sync with Lerna and recreate most of their PRs, except for the ones that I'm less interested like the one you wrote a comment about, I'm also 😱 to that Nx change, that will obviously not be pushed into Lerna-Lite. 

Anyway, if that is of no interest then disregard the post and close the issue
Cheers and happy coding 😉

**Comment by AviVahl** [comment]
Hey, thanks for reaching out.

I feel the same regarding the recent lerna changes.

That project has been in a barely maintained state for so long, that we've decided to avoid installing it every time.
My team's current usage of lerna boils down `lerna version`. CI later publishes using in-house tool named [pleb](https://github.com/wixplosives/pleb) (which executes `npm publish`). That's why we're installing it on the fly using npx upon need.

We've considered and even started working on `pleb version` to avoid needing lerna at all, but we haven't gotten around to finishing it.

I'll check lerna-lite out... might be a nice replacement for all our repositories still installing lerna.

**Comment by ghiscoding** [comment]
> We've considered and even started working on `pleb version` to avoid needing lerna at all, but we haven't gotten around to finishing it.

We never have enough tools and the best tools are often the ones we code ourselves for our own needs 😄 

> I'll check lerna-lite out... might be a nice replacement for all our repositories still installing lerna.

you should, I made Lerna-Lite a lot more modular, it only includes `version` and `publish` by default (well I should say also `info` and `init` but those are tiny) all other commands are all optional (like `run`, `exec`, `list`, ...) through extra npm install, if you don't need/want it then don't install extras. 

I guess we can close the issue if you don't want to include lerna/lerna-lite into the package.json
Have a nice one, cheers

**Comment by AviVahl** [comment]
To be honest, the most ideal solution IMO is for `npm version` to behave properly with workspaces. It currently has a few issues that render it useless without additional commands to follow.

Check out our publish flow commands for stylable, just to see what I mean:
https://github.com/wix/stylable/wiki/Publish-Flow

Thanks again for the hint. I'll be sure to check it out.

**Comment by ghiscoding** [comment]
I'm not exactly sure which step(s) exactly you meant that were missing in Lerna, but there was a recent option I added in Lerna-Lite which is `lerna version --sync-workspace-lock](https://github.com/ghiscoding/lerna-lite/tree/main/packages/version#--sync-workspace-lock) to update the lock file after the version is processed (internally it calls different command for different client, ie for npm it calls `npm install --package-lock-only`), it doesn't work with all clients (ie yarn classic, or npm < 7) hence why it's optional

Also to be honest, I'm just a regular developer and not entirely familiar with Lerna's codebase and how everything works , especially in the git world, but at the time I was just looking to update all their old deps and then I started adding couple of extra features.
